### PR TITLE
*: Add the field of history information in job structure

### PR DESCRIPTION
**Pull request by zimulala**
Handle #1936

### CODE REVIEW

**Comment by zimulala** [comment]
PTAL @GregoryIan @coocood @tiancaiamao

File: model/ddl.go [hunk 1]
```diff
@@ -73,6 +73,27 @@ func (action ActionType) String() string {
 	}
 }
 
+// HistoryInfo is used for binlog.
+type HistoryInfo struct {
+	Version   int64
```
Comment (ngaut) [inline model/ddl.go]: What does Version mean ?

Comment (zimulala) [inline model/ddl.go]: It's schema version.

**Comment by zimulala** [comment]
PTAL @ngaut

**Comment by zimulala** [comment]
This is for compatibility with previous version. 
So it is the first step #2170 , the second is remove the assignment to the job's arguments.

**Comment by tiancaiamao** [comment]
PTAL @GregoryIan

**Comment by zimulala** [comment]
PTAL @tiancaiamao @GregoryIan @coocood

**Comment by GregoryIan** [comment]
LGTM @tiancaiamao @coocood PTAL

**Comment by coocood** [comment]
LGTM

**Pull request merged**